src/entity/ReadingStreakActions.ts

```diff
@@ -0,0 +1,24 @@
+import { Column, Entity, OneToOne, PrimaryColumn } from 'typeorm';
+import { UserStreak } from './user';
+
+export enum ReadingStreakActionType {
+  Recover = 'recovered',
+}
+
+@Entity()
+export class ReadingStreakActions {
+  @PrimaryColumn({ length: 36, default: '' })
```

Comment:
Why is the default empty? you can just omit it on primary.
```suggestion
  @PrimaryColumn({ type: 'text' })
```